Classes/Domain/Model/Configuration.php

```diff
@@ -204,6 +204,7 @@ public function setAllDay($allDay)
      */
     public function getEndDate()
     {
+        $this->endDate->setTimezone(new \DateTimeZone(date_default_timezone_get()));
```

Comment:
Please use the Utility function:
https://github.com/lochmueller/calendarize/blob/master/Classes/Utility/DateTimeUtility.php#L75